includes/Core/Util/Debug_Data.php

```diff
@@ -261,6 +262,53 @@ private function get_user_status_field() {
 		);
 	}
 
+	/**
+	 * Gets the field definition for the verification_status field.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_verification_status_field() {
+		$label = __( 'Verification Status', 'google-site-kit' );
+
+		$is_verified               = $this->authentication->verification()->get();
+		$is_verified_by_file_token = $this->authentication->verification_file()->get();
+		$is_verified_by_meta_tag   = $this->authentication->verification_meta()->get();
+
+		if ( ! $is_verified ) {
+			return array(
+				'label' => $label,
+				'value' => __( 'Not verified', 'google-site-kit' ),
+				'debug' => 'not-verified',
+			);
+		}
+
+		if ( $is_verified_by_file_token ) {
+			return array(
+				'label' => $label,
+				'value' => __( 'Verified through file', 'google-site-kit' ),
+				'debug' => 'verified-file',
+			);
+		}
+
+		if ( $is_verified_by_meta_tag ) {
+			return array(
+				'label' => $label,
+				'value' => __( 'Verified through meta tag', 'google-site-kit' ),
+				'debug' => 'verified-meta',
+			);
+		}
+
+		return array(
+			'label' => $label,
+			'value' => __( 'Verified outside of Site Kit', 'google-site-kit' ),
+			'debug' => 'verified-non-site-kit',
+		);
+
+	}
```

Comment:
First PR I've done with PHP so any comments very welcome!

The developer experience looks better with PHP though than our React code.  It had real autoformatting (like prettier) so it doesn't look like there'll be any formatting feedback

Reply:
Nice job! You even put the method in the right place 😄  – which is more of a convention than anything, but it's great that you recognized it and followed the precedent 👍 👍